src/DynamoCoreWpf/Commands/DynamoCommands.cs

```diff
@@ -60,6 +60,15 @@ private bool CanInsertPausePlaybackCommand(object parameters)
             return (automationSettings.CurrentState == AutomationSettings.State.Recording);
         }
 
+        /// <summary>
+        /// Saves all recorded commands on disk (%TMP%/Commands-{0:yyyyMMdd-hhmmss}.xml)
+        /// </summary>
+        /// <returns>The path to the commands file</returns>
+        internal string DumpRecordedCommands()
+        {
+            return automationSettings.SaveRecordedCommands();
```

Comment:
are these recorded in Release builds normally?

Reply:
Commands are recorded in Release builds but only if Dynamo was started with a valid `CommandFilePath` property in StartConfiguration